docusaurus.config.js

```diff
@@ -146,16 +146,7 @@ module.exports = {
           title: "Community",
           items: [
             {
-              label: "Slack",
-              href: "https://camunda.com/slack",
-            },
-            {
-              label: "Twitter",
-              href: "https://twitter.com/camunda",
-            },
-            {
-              label: "GitHub",
-              href: "https://github.com/camunda/camunda-platform-docs",
+              html: `<a href="https://camunda.com/slack" target="_blank" rel="noreferrer noopener"><img src= "img/Slack-mark-white-RGB.png" alt="Camunda on Slack" class="footer-logos" /></a> <a href="https://twitter.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/twitter.svg" alt="Camunda on Twitter" class="footer-logos" /></a> <a href="https://github.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/github-mark-white.svg" alt="Camunda on GitHub" class="footer-logos" /></a>`,
```

Comment:
```suggestion
              html: `<a href="https://camunda.com/slack" target="_blank" rel="noreferrer noopener"><img src= "/img/Slack-mark-white-RGB.png" alt="Camunda on Slack" class="footer-logos" /></a> <a href="https://twitter.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "/img/twitter.svg" alt="Camunda on Twitter" class="footer-logos" /></a> <a href="https://github.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "/img/github-mark-white.svg" alt="Camunda on GitHub" class="footer-logos" /></a>`,
```

I'm surprised it would show up during link-checking, but bravo, robots!

Adding a `/` to the beginning of the image paths to pull from the root, instead of relative to the current page.

Reply:
Woooow!! 👏 Great catch by the robots, and thank you for the suggestion.